module/VuFind/src/VuFind/Search/Base/Params.php

```diff
@@ -970,20 +970,24 @@ public function addCheckboxFacet($filter, $desc)
      * @param string $default Default field name (null for default behavior).
      *
      * @return string         Human-readable description of field.
-     *
-     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
     public function getFacetLabel($field, $value = null, $default = null)
     {
+        $checkboxFacet = $this->checkboxFacets[$field]["$field:$value"] ?? null;
         if (!isset($this->facetConfig[$field])
             && !isset($this->extraFacetLabels[$field])
+            && null === $checkboxFacet
             && isset($this->facetAliases[$field])
         ) {
             $field = $this->facetAliases[$field];
+            $checkboxFacet = $this->checkboxFacets[$field]["$field:$value"] ?? null;
         }
         if (isset($this->facetConfig[$field])) {
             return $this->facetConfig[$field];
         }
+        if (null !== $checkboxFacet) {
```

Comment:
I think there's a question about precedence here. If we have a field active, plus a checkbox facet related to that field, which label should we use? Right now, the field is taking precedence over the checkbox, but I wonder if favoring the most specific option would be preferable. In any case, I've added a test case to #2261 to cover this situation, and it currently matches the behavior here (since that's the only way it will pass on the dev branch before this is merged) -- see commit 6585e4d37fc54a8b6547a3a4850858539a232819. If we change this, we'll need to revise that test... and in any case, we'll need to add more tests to cover situations where regular facet fields and checkbox facet fields do not overlap. Obviously that will have to wait until we can merge either this PR or #2261.

Reply:
Yes, now updated. Test revised and another added to confirm aliases work.